scripts/drug-search-bench.py

```
@@ -0,0 +1,166 @@
+
+
+def bulk_load(index, docs):
+    lines = []
+    for d in docs:
+        lines.append('{"index":{}}')
+        lines.append(json.dumps({"itemName": d["itemName"],
+                                 "itemNameChosung": d.get("itemNameChosung", "")}, ensure_ascii=False))
+    raw = ("\n".join(lines) + "\n").encode()
+    r = urllib.request.Request(f"{ES}/{index}/_bulk?refresh", method="POST", data=raw,
+                               headers={"Content-Type": "application/x-ndjson"})
+    if json.load(urllib.request.urlopen(r)).get("errors"):
```

Ruff (0.16.3)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)